packages/suir-component-mapper/README.md

```diff
@@ -0,0 +1,183 @@
+[![npm version](https://badge.fury.io/js/%40data-driven-forms%2Fmui-component-mapper.svg)](https://badge.fury.io/js/%40data-driven-forms%2Fmui-component-mapper)
+
+[![Data Driven Form logo](images/logo.png)](https://data-driven-forms.org/)
+
+Material-UI component mapper for [Data Driven Forms](https://github.com/data-driven-forms/react-forms).
```

Comment:
MUI mentions in readme

Reply:
Looking at the other mappers readme I think we should re-visit them before the 2.4 release. There is more incorrect information in others aswell (used by MIQ!)